scripts/test-zero-length-data.py

```diff
@@ -1,100 +1,198 @@
-# Author: Hubert Kario, (c) 2015
+# Author: Hubert Kario, (c) 2015,2024
```

Comment:
no updating the name in this one?

Reply:
no, I try to minimise the amount of changes :)